### ISSUE TITLE
about citation 55 in the paper.

### ISSUE THREAD

**Issue by icoz69** [CaraJ7/CoMat#5]
hi, i am interested in the gradient update part in the paper, which refers to [55]. however, i could not find it on the web. could you provide more details on this implementation? 
<img width="908" alt="image" src="https://github.com/CaraJ7/CoMat/assets/22427667/a9495825-1dcd-4064-9e8b-093497a70d50">

**Comment by CaraJ7** [comment]
Hi @icoz69 , thanks for your attention. Our training code will be released soon. Please stay tuned!

**Comment by CaraJ7** [comment]
Hi @icoz69, the citation is now available [here](https://arxiv.org/pdf/2405.00760). Check it out please.
